consensus/reward/schedule.go

```diff
@@ -18,86 +18,87 @@ var (
 	// schedule is the Token Release Schedule of Harmony
 	releasePlan = map[int64]numeric.Dec{
 		//2019
-		mustParse("2019-May-31"): numeric.MustNewDecFromStr("0.2429"),
-		mustParse("2019-Jun-30"): numeric.MustNewDecFromStr("0.2488"),
-		mustParse("2019-Jul-31"): numeric.MustNewDecFromStr("0.2547"),
-		mustParse("2019-Aug-31"): numeric.MustNewDecFromStr("0.2607"),
-		mustParse("2019-Sep-30"): numeric.MustNewDecFromStr("0.2666"),
-		mustParse("2019-Oct-31"): numeric.MustNewDecFromStr("0.2726"),
-		mustParse("2019-Nov-30"): numeric.MustNewDecFromStr("0.3785"),
-		mustParse("2019-Dec-31"): numeric.MustNewDecFromStr("0.3844"),
+		mustParse("2019-May-31"): numeric.MustNewDecFromStr("0.242864761904762"),
+		mustParse("2019-Jun-30"): numeric.MustNewDecFromStr("0.254747114845938"),
```

Comment:
This line is wrong. Did you programmatically did the change?

Reply:
Please double check. The number is critical.

Reply:
yes, programmaticaly. Multiple line copy-paste in emacs. Just I did the multiple line copy paste after that line, (initially was doing by hand on the first 2)

Reply:
ok